### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [Shikochin/ruin#1] [Comment]
# about/

我是西宮 缄，你也可以叫我 Shikochin
这是我的不知道第几个网站了，从静态到动态到静态再到动态，我就像一个流离失所的游子，没有固定的居所
不过，只要我还对此保留着兴趣，只要还有人在关注着我，你就永远可以在这里找到我
这里之所以被称为 Ruin（遗迹），完全是出于我对自己创作的内容构以的巧思，像是一些极为碎散的文字和诗篇，都在此地有所呈现

https://shikoch.in/about/

**Comment by Shikochin** [comment]
孩子们，我回来了（😭

**Reply by Keritial** [reply]
你好

**Comment by LingLambda** [comment]
大作家究竟有多少站点

**Reply by Shikochin** [reply]
并非大作家
